runtime/vm/jfr.cpp

```diff
@@ -61,6 +80,9 @@ static void jfrExecutionSampleCallback(J9VMThread *currentThread, IDATA handlerK
 static void jfrThreadCPULoadCallback(J9VMThread *currentThread, IDATA handlerKey, void *userData);
 static bool areJFRBuffersReadyForWrite(J9VMThread *currentThread);
 
+static UDATA jfrTypeIDHashFn(void *key, void *userData);
```

Comment:
Every function with a `static` forward declaration should be matched with a `static` definition (e.g. `jfrCPULoad`, `jfrThreadCPULoad`, etc.).

Reply:
Neither of those functions have forward declarations

Reply:
I've opened https://github.com/eclipse-openj9/openj9/pull/21150 for general jfr.cpp fixes.